journal.py

```diff
@@ -72,9 +73,51 @@ def list_view(request):
     return {'entries': entries}
 
 
+@view_config(route_name='detail', renderer='templates/detail.jinja2')
+def detail_view(request, session=None):
+    entry_id = request.params.get('id')
+    if session is None:
+        session = DBSession
+    entry = session.query(Entry).get(entry_id)
```

Comment:
There is a potential here that the `get` api returns None (if the `entry_id` does not correspond to a real entry, for example).  What would happen on line 82 below if that happens here?